Entity/Door/g_door.cpp

```diff
@@ -59,13 +69,48 @@ Door::Door(const int id,
         Vertex C = { glm::vec3(mapPoly.vertices[2].pos.x, 
                                mapPoly.vertices[2].pos.y, 
                                mapPoly.vertices[2].pos.z) };
+
+        float minX = glm::min( A.pos.x, B.pos.x, C.pos.x );
+        float minY = glm::min( A.pos.y, B.pos.y, C.pos.y );
+        float minZ = glm::min( A.pos.z, B.pos.z, C.pos.z );
+        float maxX = glm::max( A.pos.x, B.pos.x, C.pos.x );
+        float maxY = glm::max( A.pos.y, B.pos.y, C.pos.y );
+        float maxZ = glm::max( A.pos.z, B.pos.z, C.pos.z );
+
+        mins = glm::min( mins, glm::vec3(minX, minY, minZ) );
+        maxs = glm::max( maxs, glm::vec3(maxX, maxY, maxZ) );
+
         A.color = triColor;
         B.color = triColor;
         C.color = triColor;
         MapTri tri = { A, B, C };
 
         m_MapTris.push_back(tri);
     }
+
+    m_Mins = mins;
+    m_Maxs = maxs;
```

Comment:
My brain is too occupied to answer this correctly right now, sorry Second question is too far in the future but we need to keep maptris as they represent the CPU side geometry used for collision detection. Detecting the collision with the hkd_model won't work because it is defined in local space and therefore does not have the correct values for per triangle collision detection. 
Not sure about the first question. I think I don't quite understand it?